# timestamp information
uint64 framenumber
time timestamp
# camera information
string cam_id
# fly information
float32 x
float32 y
float32 slope
# image of fly
uint16 imbufx
uint16 imbufy
uint16 imwidth
uint16 imheight
string coding
uint8[] im
